plugins/commands/plugins/src/main/kotlin/PluginsCommand.kt

```diff
@@ -92,7 +87,9 @@ class PluginsCommand(descriptor: PluginDescriptor = PluginsCommandFactory.descri
                         bulletText = "*"
                     )
                 )
-
+                echo()
+            } else {
+                echo("Options: N/A")
```

Comment:
I'd prefer "none" as "N/A" sounds like information about plugin options is missing for some reason. But I also find it ok to just skip the section if there are no options, or to use a sentence like "This plugin does not provide any configuration options".